pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -46,20 +46,25 @@ const (
 
 //NodeLabeller struct holds informations needed to run node-labeller
 type NodeLabeller struct {
-	kvmController     *device_manager.DeviceController
-	clientset         kubecli.KubevirtClient
-	host              string
-	namespace         string
-	logger            *log.FilteredLogger
-	clusterConfig     *virtconfig.ClusterConfig
-	hypervFeatures    supportedFeatures
-	hostCapabilities  supportedFeatures
-	queue             workqueue.RateLimitingInterface
-	supportedFeatures []string
-	cpuInfo           cpuInfo
+	kvmController           *device_manager.DeviceController
+	clientset               kubecli.KubevirtClient
+	host                    string
+	namespace               string
+	logger                  *log.FilteredLogger
+	clusterConfig           *virtconfig.ClusterConfig
+	hypervFeatures          supportedFeatures
+	hostCapabilities        supportedFeatures
+	queue                   workqueue.RateLimitingInterface
+	supportedFeatures       []string
+	cpuInfo                 cpuInfo
+	loadedCPUInfo           bool
+	loadedHyperv            bool
+	loadedHostCapabilities  bool
+	loadedSupportedFeatures bool
```

Comment:
nit: What you think about renaming these to "XXIsLoaded",e.g: `CPUInfoIsLoaded`, `HyperVIsLoaded`?